### PR TITLE
.github/workflows: fix hubble installation using cilium-cli

### PR DESCRIPTION
**Pull request by aanm**
'cilium hubble enable` needs to specify the base-version that is going
to be deployed.

Fixes: 27590a95dc26 (".github: enable cilium-cli helm based installation")
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Comment by aanm** [comment]
let's wait for CI to confirm if there's no regression before merging

**Review by michi-covalent — APPROVED** [review]

**Pull request merged**
